docs/Profiles.md

```diff
@@ -1,65 +1,247 @@
 ```
+To save configuration changes:
+```
+save
+```
 
 # Rate Profiles
 
-Cleanflight supports rate profiles in addition to regular profiles.
-
-Rate profiles contain settings that adjust how your craft behaves based on control input.
+A Rate Profile is a set of Rate related settings. These profiles store the settings for Rate Type, RC Rate, Super Rate, Expo for Pitch, Roll, & Yaw.  Along with Throttle Expo and TPA settings.  
 
-Three rate profiles are supported.
+Pre-BetaFlight 3.2 Profiles and Rate Profiles are coupled together.  Each Profile had its own set of 3 Rate Profiles.  Allowing up to 9 Rate Profiles all together.
+Betaflight 3.2 broke this link, giving us independent Profiles and Rate Profiles.  Selecting Rate Profiles are independent of Profiles now.  Allowing only 3 Rate Profiles.
+BetaFlight 3.3 > the number of Rate Profiles increased from 3 to 6 Rate Profiles.  
 
-Rate profiles can be selected while flying using the inflight adjustments feature.
+## Changing Rate Profiles
+Rate Profiles can be selected using a GUI, CLI, OSD, or AUX channel. Once selected (just like Profiles), changes to settings can be made and saved to the active Rate Profile.  Also, when changing our Rate Profile that profile will remain the active Profile even over power cycles, except when using the AUX channel option.  When BetaFlight boots the active profile is selected by the position of the AUX channel.
 
-Each normal profile has a setting called 'default_rate_profile`.  When a profile is activated the
-corresponding rate profile is also activated. 
+Betaflight Configurator (GUI): Click PID Tuning Tab > Select a Rate Profile using the drop-down menu.  Once selected, any changes you make to your settings and click Save will be store to the selected Rate Profile.  The action of selecting a Rate Profile also activates that rate profile for current use.
 
-Profile 0 has a default rate profile of 0.
-Profile 1 has a default rate profile of 1.
-Profile 2 has a default rate profile of 2.
+OSD: Access CMS (Left Yaw + Pitch up) > Select `Profiles` > Change `Rate Prof`.  Make sure to Save + Exit or Save + Reboot to save settings.
 
-The defaults are set this way so that it's simple to configure a profile and a rate profile at the same.
+Aux Channel (In-flight Adjustments): Adjustments need to be configured first using the GUI.  You must have an available AUX channel configured on your radio.  
+Enable Expert Mode in the Configurator > Click Adjustments
+Click the slider for If enable.  Select the AUX # channel to use.
+For the Range, Select the whole Range from 900 to 2100.  Select Rate Profile Selection from the drop-down menu.  Use Slot 1, or any Slot not used.  Via Channel is the same as the AUX channel used.  
 
-The current rate profile can be shown or set using the CLI `rateprofile` command:
+![BetaFlight Adjustments](https://github.com/linuxdude21/betaflight/blob/master/docs/Screenshots/Betaflight%20Configurator-%20RPS%20in%20Adjustments.png?raw=true)
```

Comment:
This is now pointing to the version of the image in your fork of the repository, which isn't ideal - the file should be added to the main repository, and the link should be pointing there.

Reply:
Sorry about that, this is my first time posting to Github.  I really don't know what I'm doing.  I was trying to understand the best way to add images to the doc.  How do I add the image to the main repo?  I want to add more to this doc as other docs.  What is the best way to add Images in the future?  Sorry for the ignorance.

Thank you, Nick

Reply:
No problem. From the look of it you have added the image only to `master` of your fork. What you need to do is add it to the pull request, and it will show up in here (delete from `master`, commit, checkout branch, add, commit, push).
Then you only need to rewrite the image URL in the document to be https://github.com/betaflight/betaflight/...

Reply:
I think I added the Image to this Pull Request.  I have deleted the image in my repo (https://github.com/linuxdude21/betaflight/tree/master/docs/Screenshots).  
But I don't know how to checkout branch. Or the rest of the process.